docs/customosbuildinstructions.md

```diff
@@ -105,6 +105,12 @@ Here are the expected contents of each subdirectory /file
              /bin/iproute
              /bin/hostname
 
+    - Required binaires: utilities used by docker
+
+             /bin/ls
+             /bin/cat
+             /bin/test
```

Comment:
I was suggesting instead of "test", use other name dir_validatation_util so something more specific about its purpose.